frontend/src/Commons/Forms/InputModules/FieldsAndSoil/FieldsAndSoil.style.ts

```diff
@@ -73,5 +74,41 @@ const StyledButtonGroupContainer = styled.div`
     width: 67px;
   }
 `;
+const StyledTestContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+`;
+const StyledRadioGroupContainer = styled.div`
+  display: flex;
+
+  @media (min-width: ${screenSizes.desktop}) {
+    width: 50%;
+  }
+`;
+
+const HeaderLabel = styled.div`
+  font-weight: ${tokens.typographyFontWeightsBold};
+  margin-bottom: -10px;
```

Comment:
While negative margins are not bad practice, it might be worth looking to see if there's a non-negative way to go about that one